src/app/[locale]/layout.tsx

```diff
@@ -1,11 +1,13 @@
 import type { ReactNode } from "react";
 import type { Metadata } from "next";
-import { Plus_Jakarta_Sans, Vazirmatn } from "next/font/google";
+import { Vazirmatn } from "next/font/google";
 import { NextIntlClientProvider, useMessages } from "next-intl";
 import { unstable_setRequestLocale } from "next-intl/server";
 import { ClerkProvider } from "@clerk/nextjs";
 import { notFound } from "next/navigation";
 
+import { Outfit } from "next/font/google";
```

Comment:
Duplicate import from the same source. Please include `Outfit` with `Vazirmatn` on the top of this file